src/js/tests/utils/DateCell.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { render, screen, within } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from 'store';
+
+import DateCell from 'components/DataTable/DateCell';
+
+import '@testing-library/jest-dom';
+
+let dateCell;
```

Comment:
Is it necessary to declare dateCell as a let? As i can see you are assigning value to this variable in each "it" function, so I think you can use something like: `const dateCell = ...` in each function

Reply:
@alannadolny it is working fine, so I changed it